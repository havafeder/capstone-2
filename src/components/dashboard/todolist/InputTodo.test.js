import React from "react";
import { render } from "@testing-library/react";
import InputTodo from "./InputTodo";
import { MemoryRouter } from "react-router";

it("matches snapshot", function () {
	const { asFragment } = render(
		<MemoryRouter>
			<InputTodo />
		</MemoryRouter>,
	);
	expect(asFragment()).toMatchSnapshot();
});