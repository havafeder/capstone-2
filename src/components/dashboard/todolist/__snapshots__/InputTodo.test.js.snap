// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="d-flex"
  >
    <input
      class="form-control input"
      placeholder="add todo"
      type="text"
      value=""
    />
    <button
      class="btn add"
    >
      add
    </button>
  </form>
</DocumentFragment>
`;
